examples/stm32/f4/stm32f4-discovery/usb_cdcacm/cdcacm.c

```diff
@@ -149,11 +159,12 @@ static const struct usb_config_descriptor config = {
 	.bLength = USB_DT_CONFIGURATION_SIZE,
 	.bDescriptorType = USB_DT_CONFIGURATION,
 	.wTotalLength = 0,
-	.bNumInterfaces = 2,
+	.bNumInterfaces = sizeof(ifaces)/sizeof(ifaces[0]),
 	.bConfigurationValue = 1,
 	.iConfiguration = 0,
-	.bmAttributes = 0x80,
-	.bMaxPower = 0x32,
+	.bmAttributes = 0x80,	// D7 is reserved and must be set to one for 
+			 	// historical reasons (Table 9-10)
+	.bMaxPower = 0x32, 	// 100mA
```

Comment:
why not just make it decimal 50 instead of hex?  (I never understood why it was in hex)